Assignment-5/app/src/components/PaginationFetchedData.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from "react";
```

Comment:
This task is not as per the acceptance, on pagination data should be fetch on click of next page